### ISSUE TITLE
The Fracture causing massive lag

### ISSUE THREAD

**Issue by Carro1001** [miyo6032/MaelstromMod#21]
Once entering the Fracture dimension , it causes massive lag to my game, and i can't pinpoint why, both framerate and server-wise, to a point where my entire PC can't keep up, it appears that no other dimension has this issue, only this one, tried setting graphics to fast and had the same issues.

Here's a link to the logs: https://pastebin.com/MnF9VdhM

I tried warping to the dark nexus to get rid of the lag, but nothing responded and had to force close the game.

Maelstrom 0.3.4
forge-14.23.5.2854

**Comment by miyo6032** [comment]
Sorry that this is happening. Could you send me what mod pack and/or a list of mods you are using? I want to see if this is a conflict of some sort.

**Comment by Carro1001** [comment]
![image](https://user-images.githubusercontent.com/47877096/79890985-ad2f0000-83d6-11ea-8cda-0955ecce4f08.png)
![image](https://user-images.githubusercontent.com/47877096/79891009-b4560e00-83d6-11ea-968f-9333ad7d2f15.png)
I can also send you the actual modpack export thru twitch if you want

**Comment by miyo6032** [comment]
Yes that would be easier

**Comment by Carro1001** [comment]
[One last adventure-1.0.1.zip](https://github.com/miyo6032/MaelstromMod/files/4511503/One.last.adventure-1.0.1.zip)
Here's the export file for twitch

**Comment by miyo6032** [comment]
I tested the Fracture for your modpack for about 30 mins, and I found no major problem. I think I have a better computer (based on your graphics card is a GTX 1050), but if there was really a major problem it wouldn't matter. However, I know the dimension is playable on a 4-year old non-gaming laptop, so I'm pretty sure your computer should have no problem. So until you find some additional information about what's happening all I can give you are some recommendations.

One thing you might try is to allocate more memory. If it's a memory issue, what you will experience is a consistent and massive frame drop every 10-30 seconds or so. I think I was using around 2-4 GB while playing, so 5-6 GB would probably be safer.

Another guess I have is that some item or ability is causing lag for some reason. Maybe try to go into the dimension with some gear and items that don't have any passive abilities.

It also might have been a one-time occurrence. Especially after the chunks have been loaded, the dimension might be totally fine after a second time logging in.
